### ISSUE TITLE
I offer GDPS hosting

### ISSUE THREAD

**Issue by Mathieu57** [Cvolton/GMDprivateServer#639]
Hello! I have a dedicated server and i'm not really using it at its maximum so i offer FREE gdps hosting! I will make everything for you (installing the server & linking the game to the server & mobile version (android only))

It will be much faster than altervista or some others free hosting website

If you want to have your own server just contact me on this discord: https://discord.gg/CkMV5DV

**Comment by DarkGD123** [comment]
i prefer using a free one but thanks for the offer i might do i soon

**Comment by LBlazen** [comment]
bruh

**Comment by Catover203** [comment]
may be it a bad

1.You can remote anyonce gdps
2.You can steal password (change at registerGJAccount.php)
3.Security of server will go to 50%

**Comment by fancythedeveloper** [comment]
Literally, just sign up for free on Google Cloud and you don't have to use some other guy's VPS.

**Comment by ElectroFlameOfficial** [comment]
I agree with fancy. I personally use it to host my own GDPS and I don't have any problems with it so far.

**Comment by RobloxNationGaming** [comment]
R.I.P. GDPS Free Hosting :(
